packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -75,6 +80,10 @@ const BlockDraggableWrapper = ( { children } ) => {
 
 	const { scrollRef } = useBlockListContext();
 	const animatedScrollRef = useAnimatedRef();
+	const { left, right } = useSafeAreaInsets();
+	const { width } = useSafeAreaFrame();
+	const safeAreaOffset = left + right;
+	const maxWidth = width - safeAreaOffset;
```

Comment:
I understand that this value is the content width (i.e. the screen width subtracting the safe area), right? If so, we could rename the constant to be more accurate with the value, maybe something like `contentWidth`, wdyt?

Reply:
Good idea, I'll rename it! 👍